src/modelfree/train.py

```diff
@@ -279,6 +294,10 @@ def single_wrappers(single_venv, scheduler, our_idx, log_callbacks, save_callbac
         log_callbacks.append(lambda logger, locals, globals: rew_shape_venv.log_callback(logger))
         single_venv = rew_shape_venv
 
+    for anneal_type in ['noise', 'rew_shape']:
+        if scheduler.is_conditional(anneal_type):
+            scheduler.set_annealer_shaping_env(anneal_type, single_venv)
```

Comment:
I think this should be `rew_shape_venv` not `single_venv`. (They're the same here but `single_venv` could easily get wrapped again and then this wouldn't work.)